doc/Install_CORTX_on_Supermicro_Servers_with_Seagate_RBOD_Storage.md

```diff
@@ -0,0 +1,469 @@
+# Install CORTX on Supermicro Servers with Seagate RBOD Storage
+### LyveLab Israel - Bari Arviv
+
+1. Verify that the connectivity of the hardware is the same as described below: 
+   <p align="center">
+	 <img src="./images/LDRv1_connections_diagram.png">
+   </p>
+
+1. HW spec that was used for the installation:
+    - 2x Supermicro 6019U-TRT 
+    - CPU – 2x Intel Silver 4210R, 10 cores/20 threads
+    - Memory – 192GB, 12x 16GB DDR4 2666 MHz 
+    - HDD – 2x Seagate 1TB SATA HDD 
+    - Power supply – 2x 750W, redundant 
+    - Network (onboard) – 2x 1GbE/10GbE RJ-45 network interfaces (only one connected; serving BMC and Ethernet) 
+    - Network (PCI) – 1x Mellanox Connect-X5 EN (MCX516A-GCAT), two 50 GbE ports; QSFP-28 
+    - BMC – 1x BMC interface (RJ-45) 
+    - SAS HBA – LSI 9305-16e, quad-port, PCIe Gen 3 x8 
+    - 1x EXOS X 5U84 enclosure with 2x Gallium NX3 controllers (for 5U84) 
+    - 84x 16TB NL-SAS 7k RPM HDDs 
+    - 6x Ethernet cables 
+    - 8x SAS cables (note: Beta units have only 4x) 
+    - 1x break-out cable for Mellanox (QSFP-28) 
+    - 1x Mellanox cable (for direct connectl QSFP-28)
+
+1. Connect to 5U84’s web interface, go to `System->Actions->Firmware update`, and verify that bundle version is `GN280R008-04`. \
+   Other versions could work by storage level but might be unsupported by CORTX hardware monitoring features.
+
+1. Configure **both servers** identically to boot in UEFI mode. Controller configuration should be AHCI (non-RAID), \
+   boot from first local drive.
+
+1. Install [`cortx-os-1.0.0-23.iso`](https://cortxpublic.s3.us-west-2.amazonaws.com/cortx-os-1.0.0-23.iso) **on both servers**. Possible installation options are using a physical USB drive (with installation iso deployed) using PXE environment configured in UEFI mode, using kickstart file `ks.cfg` from the iso. It should be slightly modified to be run in isolated mode (non-production environment). Example configuration:
+   ```
+   [root@rack-b-node-1-gui ~]# cat /var/ftp/pub/ks.cfg
```

Comment:
What instruction are you giving here to the user? Are you asking the user to make sure that the file `/var/ftp/pub/ks.cfg` contains all of this text below or are you asking them to copy/paste this text into this file? What exactly is this instruction saying?

Reply:
Also if it were possible to make this much smaller that would be nice this is such a large piece of text...but obviously if that's not possible no problem

Reply:
The file needs to be modified, this is not a copy of the text.  
I removed the path so as not to confuse the user, for each user the path will be different.